src/PresentationalComponents/SystemsTable/Components/NoSystemsTable.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {
+  EmptyStateBody,
+  EmptyState,
+  EmptyStateVariant,
+  Text,
+  TextContent,
+  TextVariants,
+  Title,
+  Bullseye,
+} from '@patternfly/react-core';
+
+const NoSystemsTable = () => (
+  <Bullseye>
+    <EmptyState variant={EmptyStateVariant.full}>
+      <Title headingLevel="h2" size="lg" style={{ fontWeight: 'bold' }}>
+        No matching systems found
```

Comment:
Should we start adding these texts to internationalizatio (**intl.formatMessage()**)

Reply:
Yeah, let's start adding them to translations. I'll pull out the strings where I find them for this PR.